apps/music/js/ui/views/list_view.js

```diff
@@ -95,6 +95,8 @@ var ListView = {
       headerLi = document.createElement('li');
       headerLi.className = 'list-header';
       headerLi.textContent = this.lastFirstLetter || '?';
+      headerLi.setAttribute('role', 'heading');
+      headerLi.setAttribute('aria-level', '2');
```

Comment:
Do you mean the gaia-subheader? That needs to be given a role as well, but i think it is outside the scope of this bug.